lib/graphql/backend/event/fragment/event_feature.graphql

```diff
@@ -0,0 +1,8 @@
+# import '../../schema.graphql';
+
+fragment EventFeature on Feature {
+  id: _id
```

Comment:
what if we query these inside event detail ? this will cause conflict with EventDto where the properties still looking for snake_case ones 
Be careful when using with Event entities 
not recommend to transform at this time